napari/_qt/qt_viewer.py

```diff
@@ -155,41 +132,54 @@ class QtViewer(QSplitter):
     show_welcome_screen : bool, optional
         Flag to show a welcome message when no layers are present in the
         canvas. Default is `False`.
+    canvas_class : napari._vispy.canvas.VispyCanvas
+        The VispyCanvas class providing the Vispy SceneCanvas. Users can also
+        have a custom canvas here.
 
     Attributes
     ----------
-    canvas : vispy.scene.SceneCanvas
-        Canvas for rendering the current view.
-    console : QtConsole
-        IPython console terminal integrated into the napari GUI.
-    controls : QtLayerControlsContainer
-        Qt view for GUI controls.
+    canvas : napari._vispy.canvas.VispyCanvas
+        The VispyCanvas class providing the Vispy SceneCanvas. Users can also
+        have a custom canvas here.
     dims : napari.qt_dims.QtDims
         Dimension sliders; Qt View for Dims model.
-    dockConsole : QtViewerDockWidget
+    show_welcome_screen : bool
+        Boolean indicating whether to show the welcome screen.
+    viewer : napari.components.ViewerModel
+        Napari viewer containing the rendered scene, layers, and controls.
+    _console : napari_console.QtConsole
+        IPython console terminal integrated into the napari GUI.
+    _controls : napari._qt.layer_controls.QtLayerControlsContainer
+        Qt view for GUI controls.
+    _dockConsole : napari._qt.widgets.qt_viewer_dock_widget.QtViewerDockWidget
         QWidget wrapped in a QDockWidget with forwarded viewer events.
-    dockLayerControls : QtViewerDockWidget
+    _dockLayerControls : napari._qt.widgets.qt_viewer_dock_widget.QtViewerDockWidget
         QWidget wrapped in a QDockWidget with forwarded viewer events.
-    dockLayerList : QtViewerDockWidget
+    _dockLayerList : napari._qt.widgets.qt_viewer_dock_widget.QtViewerDockWidget
         QWidget wrapped in a QDockWidget with forwarded viewer events.
```

Comment:
All these attributes have public getters without `_` prefix. Is the reason not to document these getters?